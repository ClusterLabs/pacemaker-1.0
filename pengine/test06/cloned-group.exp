 <transition_graph cluster-delay="60s" stonith-timeout="60s" failed-stop-offset="INFINITY" failed-start-offset="INFINITY" batch-limit="30" transition_id="0">
   <synapse id="0">
     <action_set>
       <pseudo_event id="13" operation="start" operation_key="grrr:0_start_0">
         <attributes CRM_meta_clone="0" CRM_meta_clone_max="2" CRM_meta_clone_node_max="1" CRM_meta_timeout="20000" crm_feature_set="3.0.1"/>
       </pseudo_event>
     </action_set>
     <inputs>
       <trigger>
         <pseudo_event id="15" operation="stop" operation_key="grrr:0_stop_0"/>
       </trigger>
       <trigger>
         <pseudo_event id="16" operation="stopped" operation_key="grrr:0_stopped_0"/>
       </trigger>
       <trigger>
         <pseudo_event id="23" operation="start" operation_key="apache2_clone_start_0"/>
       </trigger>
     </inputs>
   </synapse>
   <synapse id="1">
     <action_set>
       <pseudo_event id="14" operation="running" operation_key="grrr:0_running_0">
         <attributes CRM_meta_clone="0" CRM_meta_clone_max="2" CRM_meta_clone_node_max="1" CRM_meta_timeout="20000" crm_feature_set="3.0.1"/>
       </pseudo_event>
     </action_set>
     <inputs>
       <trigger>
         <rsc_op id="7" operation="start" operation_key="apache2:0_start_0" on_node="webcluster01" on_node_uuid="49e81295-8e2f-4aeb-98f3-a14de6f62298"/>
       </trigger>
       <trigger>
         <rsc_op id="11" operation="start" operation_key="mysql-proxy:0_start_0" on_node="webcluster01" on_node_uuid="49e81295-8e2f-4aeb-98f3-a14de6f62298"/>
       </trigger>
       <trigger>
         <pseudo_event id="13" operation="start" operation_key="grrr:0_start_0"/>
       </trigger>
     </inputs>
   </synapse>
   <synapse id="2">
     <action_set>
       <pseudo_event id="15" operation="stop" operation_key="grrr:0_stop_0">
         <attributes CRM_meta_clone="0" CRM_meta_clone_max="2" CRM_meta_clone_node_max="1" CRM_meta_timeout="20000" crm_feature_set="3.0.1"/>
       </pseudo_event>
     </action_set>
     <inputs>
       <trigger>
         <pseudo_event id="25" operation="stop" operation_key="apache2_clone_stop_0"/>
       </trigger>
     </inputs>
   </synapse>
   <synapse id="3">
     <action_set>
       <pseudo_event id="16" operation="stopped" operation_key="grrr:0_stopped_0">
         <attributes CRM_meta_clone="0" CRM_meta_clone_max="2" CRM_meta_clone_node_max="1" CRM_meta_timeout="20000" crm_feature_set="3.0.1"/>
       </pseudo_event>
     </action_set>
     <inputs>
       <trigger>
         <rsc_op id="10" operation="stop" operation_key="apache2:0_stop_0" on_node="webcluster01" on_node_uuid="49e81295-8e2f-4aeb-98f3-a14de6f62298"/>
       </trigger>
       <trigger>
         <rsc_op id="12" operation="stop" operation_key="mysql-proxy:0_stop_0" on_node="webcluster01" on_node_uuid="49e81295-8e2f-4aeb-98f3-a14de6f62298"/>
       </trigger>
       <trigger>
         <pseudo_event id="15" operation="stop" operation_key="grrr:0_stop_0"/>
       </trigger>
     </inputs>
   </synapse>
   <synapse id="4">
     <action_set>
       <rsc_op id="4" operation="monitor" operation_key="apache2:0_monitor_10000" on_node="webcluster01" on_node_uuid="49e81295-8e2f-4aeb-98f3-a14de6f62298">
         <primitive id="apache2:0" long-id="apache2_clone:grrr:0:apache2:0" class="ocf" provider="heartbeat" type="apache"/>
         <attributes CRM_meta_clone="0" CRM_meta_clone_max="2" CRM_meta_clone_node_max="1" CRM_meta_interval="10000" CRM_meta_name="monitor" CRM_meta_timeout="20000" configfile="/etc/apache2/apache2.conf" crm_feature_set="3.0.1" httpd="/usr/sbin/apache2" options="-k start" target_role="started"/>
       </rsc_op>
     </action_set>
     <inputs>
       <trigger>
         <rsc_op id="7" operation="start" operation_key="apache2:0_start_0" on_node="webcluster01" on_node_uuid="49e81295-8e2f-4aeb-98f3-a14de6f62298"/>
       </trigger>
     </inputs>
   </synapse>
   <synapse id="5">
     <action_set>
       <rsc_op id="7" operation="start" operation_key="apache2:0_start_0" on_node="webcluster01" on_node_uuid="49e81295-8e2f-4aeb-98f3-a14de6f62298">
         <primitive id="apache2:0" long-id="apache2_clone:grrr:0:apache2:0" class="ocf" provider="heartbeat" type="apache"/>
         <attributes CRM_meta_clone="0" CRM_meta_clone_max="2" CRM_meta_clone_node_max="1" CRM_meta_timeout="20000" configfile="/etc/apache2/apache2.conf" crm_feature_set="3.0.1" httpd="/usr/sbin/apache2" options="-k start" target_role="started"/>
       </rsc_op>
     </action_set>
     <inputs>
       <trigger>
         <rsc_op id="10" operation="stop" operation_key="apache2:0_stop_0" on_node="webcluster01" on_node_uuid="49e81295-8e2f-4aeb-98f3-a14de6f62298"/>
       </trigger>
       <trigger>
         <pseudo_event id="13" operation="start" operation_key="grrr:0_start_0"/>
       </trigger>
     </inputs>
   </synapse>
   <synapse id="6">
     <action_set>
       <rsc_op id="10" operation="stop" operation_key="apache2:0_stop_0" on_node="webcluster01" on_node_uuid="49e81295-8e2f-4aeb-98f3-a14de6f62298">
         <primitive id="apache2:0" long-id="apache2_clone:grrr:0:apache2:0" class="ocf" provider="heartbeat" type="apache"/>
         <attributes CRM_meta_clone="0" CRM_meta_clone_max="2" CRM_meta_clone_node_max="1" CRM_meta_timeout="20000" crm_feature_set="3.0.1"/>
       </rsc_op>
     </action_set>
     <inputs>
       <trigger>
         <rsc_op id="12" operation="stop" operation_key="mysql-proxy:0_stop_0" on_node="webcluster01" on_node_uuid="49e81295-8e2f-4aeb-98f3-a14de6f62298"/>
       </trigger>
       <trigger>
         <pseudo_event id="15" operation="stop" operation_key="grrr:0_stop_0"/>
       </trigger>
     </inputs>
   </synapse>
   <synapse id="7">
     <action_set>
       <rsc_op id="2" operation="monitor" operation_key="mysql-proxy:0_monitor_10000" on_node="webcluster01" on_node_uuid="49e81295-8e2f-4aeb-98f3-a14de6f62298">
         <primitive id="mysql-proxy:0" long-id="apache2_clone:grrr:0:mysql-proxy:0" class="lsb" type="mysql-proxy"/>
         <attributes CRM_meta_clone="0" CRM_meta_clone_max="2" CRM_meta_clone_node_max="1" CRM_meta_interval="10000" CRM_meta_name="monitor" CRM_meta_timeout="20000" crm_feature_set="3.0.1"/>
       </rsc_op>
     </action_set>
     <inputs>
       <trigger>
         <rsc_op id="11" operation="start" operation_key="mysql-proxy:0_start_0" on_node="webcluster01" on_node_uuid="49e81295-8e2f-4aeb-98f3-a14de6f62298"/>
       </trigger>
     </inputs>
   </synapse>
   <synapse id="8">
     <action_set>
       <rsc_op id="11" operation="start" operation_key="mysql-proxy:0_start_0" on_node="webcluster01" on_node_uuid="49e81295-8e2f-4aeb-98f3-a14de6f62298">
         <primitive id="mysql-proxy:0" long-id="apache2_clone:grrr:0:mysql-proxy:0" class="lsb" type="mysql-proxy"/>
         <attributes CRM_meta_clone="0" CRM_meta_clone_max="2" CRM_meta_clone_node_max="1" CRM_meta_timeout="20000" crm_feature_set="3.0.1"/>
       </rsc_op>
     </action_set>
     <inputs>
       <trigger>
         <rsc_op id="7" operation="start" operation_key="apache2:0_start_0" on_node="webcluster01" on_node_uuid="49e81295-8e2f-4aeb-98f3-a14de6f62298"/>
       </trigger>
       <trigger>
         <rsc_op id="12" operation="stop" operation_key="mysql-proxy:0_stop_0" on_node="webcluster01" on_node_uuid="49e81295-8e2f-4aeb-98f3-a14de6f62298"/>
       </trigger>
       <trigger>
         <pseudo_event id="13" operation="start" operation_key="grrr:0_start_0"/>
       </trigger>
     </inputs>
   </synapse>
   <synapse id="9">
     <action_set>
       <rsc_op id="12" operation="stop" operation_key="mysql-proxy:0_stop_0" on_node="webcluster01" on_node_uuid="49e81295-8e2f-4aeb-98f3-a14de6f62298">
         <primitive id="mysql-proxy:0" long-id="apache2_clone:grrr:0:mysql-proxy:0" class="lsb" type="mysql-proxy"/>
         <attributes CRM_meta_clone="0" CRM_meta_clone_max="2" CRM_meta_clone_node_max="1" CRM_meta_timeout="20000" crm_feature_set="3.0.1"/>
       </rsc_op>
     </action_set>
     <inputs>
       <trigger>
         <pseudo_event id="15" operation="stop" operation_key="grrr:0_stop_0"/>
       </trigger>
     </inputs>
   </synapse>
   <synapse id="10">
     <action_set>
       <pseudo_event id="21" operation="stop" operation_key="grrr:1_stop_0">
         <attributes CRM_meta_clone="1" CRM_meta_clone_max="2" CRM_meta_clone_node_max="1" CRM_meta_timeout="20000" crm_feature_set="3.0.1"/>
       </pseudo_event>
     </action_set>
     <inputs>
       <trigger>
         <pseudo_event id="25" operation="stop" operation_key="apache2_clone_stop_0"/>
       </trigger>
     </inputs>
   </synapse>
   <synapse id="11">
     <action_set>
       <pseudo_event id="22" operation="stopped" operation_key="grrr:1_stopped_0">
         <attributes CRM_meta_clone="1" CRM_meta_clone_max="2" CRM_meta_clone_node_max="1" CRM_meta_timeout="20000" crm_feature_set="3.0.1"/>
       </pseudo_event>
     </action_set>
     <inputs>
       <trigger>
         <rsc_op id="17" operation="stop" operation_key="apache2:1_stop_0" on_node="webcluster01" on_node_uuid="49e81295-8e2f-4aeb-98f3-a14de6f62298"/>
       </trigger>
       <trigger>
         <rsc_op id="18" operation="stop" operation_key="mysql-proxy:1_stop_0" on_node="webcluster01" on_node_uuid="49e81295-8e2f-4aeb-98f3-a14de6f62298"/>
       </trigger>
       <trigger>
         <pseudo_event id="21" operation="stop" operation_key="grrr:1_stop_0"/>
       </trigger>
     </inputs>
   </synapse>
   <synapse id="12">
     <action_set>
       <rsc_op id="17" operation="stop" operation_key="apache2:1_stop_0" on_node="webcluster01" on_node_uuid="49e81295-8e2f-4aeb-98f3-a14de6f62298">
         <primitive id="apache2:1" long-id="apache2_clone:grrr:1:apache2:1" class="ocf" provider="heartbeat" type="apache"/>
         <attributes CRM_meta_clone="1" CRM_meta_clone_max="2" CRM_meta_clone_node_max="1" CRM_meta_timeout="20000" crm_feature_set="3.0.1"/>
       </rsc_op>
     </action_set>
     <inputs>
       <trigger>
         <rsc_op id="18" operation="stop" operation_key="mysql-proxy:1_stop_0" on_node="webcluster01" on_node_uuid="49e81295-8e2f-4aeb-98f3-a14de6f62298"/>
       </trigger>
       <trigger>
         <pseudo_event id="21" operation="stop" operation_key="grrr:1_stop_0"/>
       </trigger>
     </inputs>
   </synapse>
   <synapse id="13">
     <action_set>
       <rsc_op id="18" operation="stop" operation_key="mysql-proxy:1_stop_0" on_node="webcluster01" on_node_uuid="49e81295-8e2f-4aeb-98f3-a14de6f62298">
         <primitive id="mysql-proxy:1" long-id="apache2_clone:grrr:1:mysql-proxy:1" class="lsb" type="mysql-proxy"/>
         <attributes CRM_meta_clone="1" CRM_meta_clone_max="2" CRM_meta_clone_node_max="1" CRM_meta_timeout="20000" crm_feature_set="3.0.1"/>
       </rsc_op>
     </action_set>
     <inputs>
       <trigger>
         <pseudo_event id="21" operation="stop" operation_key="grrr:1_stop_0"/>
       </trigger>
     </inputs>
   </synapse>
   <synapse id="14">
     <action_set>
       <pseudo_event id="23" operation="start" operation_key="apache2_clone_start_0">
         <attributes CRM_meta_clone_max="2" CRM_meta_clone_node_max="1" CRM_meta_timeout="20000" crm_feature_set="3.0.1"/>
       </pseudo_event>
     </action_set>
     <inputs>
       <trigger>
         <pseudo_event id="25" operation="stop" operation_key="apache2_clone_stop_0"/>
       </trigger>
       <trigger>
         <pseudo_event id="26" operation="stopped" operation_key="apache2_clone_stopped_0"/>
       </trigger>
     </inputs>
   </synapse>
   <synapse id="15" priority="1000000">
     <action_set>
       <pseudo_event id="24" operation="running" operation_key="apache2_clone_running_0">
         <attributes CRM_meta_clone_max="2" CRM_meta_clone_node_max="1" CRM_meta_timeout="20000" crm_feature_set="3.0.1"/>
       </pseudo_event>
     </action_set>
     <inputs>
       <trigger>
         <pseudo_event id="14" operation="running" operation_key="grrr:0_running_0"/>
       </trigger>
       <trigger>
         <pseudo_event id="23" operation="start" operation_key="apache2_clone_start_0"/>
       </trigger>
     </inputs>
   </synapse>
   <synapse id="16">
     <action_set>
       <pseudo_event id="25" operation="stop" operation_key="apache2_clone_stop_0">
         <attributes CRM_meta_clone_max="2" CRM_meta_clone_node_max="1" CRM_meta_timeout="20000" crm_feature_set="3.0.1"/>
       </pseudo_event>
     </action_set>
     <inputs/>
   </synapse>
   <synapse id="17" priority="1000000">
     <action_set>
       <pseudo_event id="26" operation="stopped" operation_key="apache2_clone_stopped_0">
         <attributes CRM_meta_clone_max="2" CRM_meta_clone_node_max="1" CRM_meta_timeout="20000" crm_feature_set="3.0.1"/>
       </pseudo_event>
     </action_set>
     <inputs>
       <trigger>
         <pseudo_event id="16" operation="stopped" operation_key="grrr:0_stopped_0"/>
       </trigger>
       <trigger>
         <pseudo_event id="22" operation="stopped" operation_key="grrr:1_stopped_0"/>
       </trigger>
       <trigger>
         <pseudo_event id="25" operation="stop" operation_key="apache2_clone_stop_0"/>
       </trigger>
     </inputs>
   </synapse>
   <synapse id="18">
     <action_set>
       <pseudo_event id="5" operation="all_stopped" operation_key="all_stopped">
         <attributes crm_feature_set="3.0.1"/>
       </pseudo_event>
     </action_set>
     <inputs>
       <trigger>
         <rsc_op id="10" operation="stop" operation_key="apache2:0_stop_0" on_node="webcluster01" on_node_uuid="49e81295-8e2f-4aeb-98f3-a14de6f62298"/>
       </trigger>
       <trigger>
         <rsc_op id="12" operation="stop" operation_key="mysql-proxy:0_stop_0" on_node="webcluster01" on_node_uuid="49e81295-8e2f-4aeb-98f3-a14de6f62298"/>
       </trigger>
       <trigger>
         <rsc_op id="17" operation="stop" operation_key="apache2:1_stop_0" on_node="webcluster01" on_node_uuid="49e81295-8e2f-4aeb-98f3-a14de6f62298"/>
       </trigger>
       <trigger>
         <rsc_op id="18" operation="stop" operation_key="mysql-proxy:1_stop_0" on_node="webcluster01" on_node_uuid="49e81295-8e2f-4aeb-98f3-a14de6f62298"/>
       </trigger>
     </inputs>
   </synapse>
 </transition_graph>

