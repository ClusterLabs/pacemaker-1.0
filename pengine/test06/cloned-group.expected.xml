<?xml version="1.0" encoding="UTF-8"?>
<cib admin_epoch="0" epoch="300" num_updates="42019" dc-uuid="49e81295-8e2f-4aeb-98f3-a14de6f62298" have-quorum="true" remote-tls-port="0" validate-with="pacemaker-1.0">
   <configuration>
     <crm_config>
       <cluster_property_set id="default"><nvpair id="nvpair" name="symmetric-cluster" value="true"/><nvpair id="nvpair" name="transition-idle-timeout" value="60"/><nvpair id="nvpair" name="default-resource-stickiness" value="0"/><nvpair id="nvpair" name="short-resource-names" value="true"/><nvpair id="nvpair" name="no-quorum-policy" value="stop"/><nvpair id="nvpair" name="stop-orphan-resources" value="false"/><nvpair id="nvpair" name="stop-orphan-actions" value="true"/><nvpair id="nvpair" name="remove-after-stop" value="false"/><nvpair id="nvpair" name="is-managed-default" value="true"/><nvpair id="nvpair" name="stonith-enabled" value="false"/><nvpair id="nvpair" name="stonith-action" value="reboot"/></cluster_property_set>
     </crm_config>
     <nodes>
       <node id="49e81295-8e2f-4aeb-98f3-a14de6f62298" uname="webcluster01" type="normal"/>
       <node id="989edcb8-aa41-4aab-9fca-65569015499d" uname="webcluster02" type="normal"/>
     </nodes>
     <resources>
       <clone id="apache2_clone"><meta_attributes id="clone-apache2_clone.meta"><nvpair id="nvpair" name="target-role" value="started"/><nvpair id="nvpair" name="clone-max" value="2"/><nvpair id="nvpair" name="clone-node-max" value="1"/></meta_attributes><group id="grrr"><meta_attributes id="group-grrr.meta"/>
           <primitive id="apache2" class="ocf" type="apache" provider="heartbeat"><meta_attributes id="primitive-apache2.meta"><nvpair id="nvpair" name="target-role" value="started"/></meta_attributes>
             <operations>
               <op name="monitor" interval="10s" id="apache2_mon" timeout="20s"/>
             </operations>
             <instance_attributes id="instance_attributes"><nvpair id="nvpair" name="httpd" value="/usr/sbin/apache2"/><nvpair id="nvpair" name="options" value="-k start"/><nvpair id="nvpair" name="configfile" value="/etc/apache2/apache2.conf"/><nvpair id="nvpair" name="target_role" value="started"/></instance_attributes>
           </primitive>
           <primitive id="mysql-proxy" class="lsb" type="mysql-proxy"><meta_attributes id="primitive-mysql-proxy.meta"/>
             <operations>
               <op name="monitor" interval="10s" id="mysql-proxy_mon" timeout="20s"/>
             </operations>
           </primitive>
         </group></clone>
     </resources>
     <constraints/>
   </configuration>
   <status>
     <node_state id="49e81295-8e2f-4aeb-98f3-a14de6f62298" uname="webcluster01" crmd="online" shutdown="0" in_ccm="true" ha="active" join="member" expected="member">
       <lrm id="lrm.auto-1">
         <lrm_resources id="lrm_resources">
           <lrm_resource id="mysql-proxy:1" type="mysql-proxy" class="lsb">
             <lrm_rsc_op id="mysql-proxy-1_monitor_0" operation="monitor" transition-key="6:0:9b0d1af2-f932-497b-971f-70be4a277e3d" transition-magic="0:7;6:0:9b0d1af2-f932-497b-971f-70be4a277e3d" call-id="5" crm_feature_set="2.0" rc-code="7" op-status="0" interval="0" op-digest="f2317cad3d54cec5d7d7aa7d0bf35cf8"/>
             <lrm_rsc_op id="mysql-proxy-1_start_0" operation="start" transition-key="15:1:9b0d1af2-f932-497b-971f-70be4a277e3d" transition-magic="0:0;15:1:9b0d1af2-f932-497b-971f-70be4a277e3d" call-id="13" crm_feature_set="2.0" rc-code="0" op-status="0" interval="0" op-digest="f2317cad3d54cec5d7d7aa7d0bf35cf8"/>
             <lrm_rsc_op id="mysql-proxy-1_monitor_10000" operation="monitor" transition-key="16:1:9b0d1af2-f932-497b-971f-70be4a277e3d" transition-magic="0:0;16:1:9b0d1af2-f932-497b-971f-70be4a277e3d" call-id="15" crm_feature_set="2.0" rc-code="0" op-status="0" interval="10000" op-digest="5811cef7f7f94e3a35a70be7916cb2fd"/>
           </lrm_resource>
           <lrm_resource id="mysql-proxy:0" type="mysql-proxy" class="lsb">
             <lrm_rsc_op id="mysql-proxy-0_monitor_0" operation="monitor" transition-key="4:0:9b0d1af2-f932-497b-971f-70be4a277e3d" transition-magic="0:7;4:0:9b0d1af2-f932-497b-971f-70be4a277e3d" call-id="3" crm_feature_set="2.0" rc-code="7" op-status="0" interval="0" op-digest="f2317cad3d54cec5d7d7aa7d0bf35cf8"/>
             <lrm_rsc_op id="mysql-proxy-0_start_0" operation="start" transition-key="6:1:9b0d1af2-f932-497b-971f-70be4a277e3d" transition-magic="0:0;6:1:9b0d1af2-f932-497b-971f-70be4a277e3d" call-id="11" crm_feature_set="2.0" rc-code="0" op-status="0" interval="0" op-digest="f2317cad3d54cec5d7d7aa7d0bf35cf8"/>
             <lrm_rsc_op id="mysql-proxy-0_monitor_10000" operation="monitor" transition-key="7:1:9b0d1af2-f932-497b-971f-70be4a277e3d" transition-magic="0:0;7:1:9b0d1af2-f932-497b-971f-70be4a277e3d" call-id="14" crm_feature_set="2.0" rc-code="0" op-status="0" interval="10000" op-digest="5811cef7f7f94e3a35a70be7916cb2fd"/>
           </lrm_resource>
           <lrm_resource id="apache2:1" type="apache" class="ocf" provider="heartbeat">
             <lrm_rsc_op id="apache2-1_monitor_0" operation="monitor" transition-key="5:0:9b0d1af2-f932-497b-971f-70be4a277e3d" transition-magic="0:7;5:0:9b0d1af2-f932-497b-971f-70be4a277e3d" call-id="4" crm_feature_set="2.0" rc-code="7" op-status="0" interval="0" op-digest="6f4f9083db544b5d5d246a882975fb04"/>
             <lrm_rsc_op id="apache2-1_start_0" operation="start" transition-key="13:1:9b0d1af2-f932-497b-971f-70be4a277e3d" transition-magic="0:0;13:1:9b0d1af2-f932-497b-971f-70be4a277e3d" call-id="9" crm_feature_set="2.0" rc-code="0" op-status="0" interval="0" op-digest="5f4f9083db544b5d5d246a882975fb04"/>
             <lrm_rsc_op id="apache2-1_monitor_10000" operation="monitor" transition-key="14:1:9b0d1af2-f932-497b-971f-70be4a277e3d" transition-magic="0:0;14:1:9b0d1af2-f932-497b-971f-70be4a277e3d" call-id="12" crm_feature_set="2.0" rc-code="0" op-status="0" interval="10000" op-digest="5fd4bafc12dc00bd492080cc8f450fcf"/>
           </lrm_resource>
           <lrm_resource id="apache2:0" type="apache" class="ocf" provider="heartbeat">
             <lrm_rsc_op id="apache2-0_monitor_0" operation="monitor" transition-key="3:0:9b0d1af2-f932-497b-971f-70be4a277e3d" transition-magic="0:7;3:0:9b0d1af2-f932-497b-971f-70be4a277e3d" call-id="2" crm_feature_set="2.0" rc-code="7" op-status="0" interval="0" op-digest="6f4f9083db544b5d5d246a882975fb04"/>
             <lrm_rsc_op id="apache2-0_start_0" operation="start" transition-key="4:1:9b0d1af2-f932-497b-971f-70be4a277e3d" transition-magic="0:0;4:1:9b0d1af2-f932-497b-971f-70be4a277e3d" call-id="8" crm_feature_set="2.0" rc-code="0" op-status="0" interval="0" op-digest="5f4f9083db544b5d5d246a882975fb04"/>
             <lrm_rsc_op id="apache2-0_monitor_10000" operation="monitor" transition-key="5:1:9b0d1af2-f932-497b-971f-70be4a277e3d" transition-magic="0:0;5:1:9b0d1af2-f932-497b-971f-70be4a277e3d" call-id="10" crm_feature_set="2.0" rc-code="0" op-status="0" interval="10000" op-digest="5fd4bafc12dc00bd492080cc8f450fcf"/>
           </lrm_resource>
         </lrm_resources>
       </lrm>
       <transient_attributes id="transient_attributes.auto-1">
         <instance_attributes id="instance_attributes"><nvpair id="nvpair" name="probe_complete" value="true"/></instance_attributes>
       </transient_attributes>
     </node_state>
   </status>
 </cib>
